src/core/utils/misc.cpp

```diff
@@ -226,13 +226,30 @@ QPoint Utils::Misc::screenCenter(QWidget *win)
 #endif
 
 /**
- * Detects the version of python by calling
- * "python --version" and parsing the output.
+ * Detects the python version.
  */
 int Utils::Misc::pythonVersion()
 {
     static int version = -1;
     if (version < 0) {
+        QString version_complete = pythonVersionComplete().trimmed();
+        QStringList splitted = version_complete.split('.');
+        if (splitted.size() > 1) {
+            int highVer = splitted.at(0).toInt();
+            if (highVer == 2 or highVer == 3)
```

Comment:
I read that "or" is indeed a valid alternative syntax for logical OR. However, msvc2013 doesn't recognize it.